CharlieBackend.Api.UnitTes/StudentGroupServiceTest.cs

```diff
@@ -121,9 +121,6 @@ public async Task CreateStudentGroupAsync_StudentGroup_ShouldReturnStudentGroup(
             
             //Assert
             successResult.Data.Should().NotBeNull();
-            successResult.Data.Name.Should().Be(newStudentGroup.Name);
-            successResult.Data.CourseId.Should().Be(newStudentGroup.CourseId);
-            successResult.Data.StudentIds.Should().BeEquivalentTo(newStudentGroup.StudentIds);
```

Comment:
I meant to replace that verification to this: successResult.Data.Should().BeEquivalentTo(newStudentGroup);

Reply:
got it fixed